<launch>
  <rosparam command="load" file="$(find wbc_pr2_ctrl)/config/pr2_left_arm_wbc.yaml" />
  <node
      pkg="wbc_pr2_ctrl"
      type="dynamic_model"
      name="dynamic_model"
      args=""
      output="screen">
  </node>
</launch>
